openfeature-provider/java/src/main/java/com/spotify/confidence/RemoteResolver.java

```diff
@@ -4,6 +4,8 @@
 import com.spotify.confidence.flags.resolver.v1.ResolveFlagsResponse;
 import java.util.concurrent.CompletableFuture;
 
-public non-sealed interface ResolverFallback extends StickyResolveStrategy {
+public interface RemoteResolver {
```

Comment:
Do we need this interface? I don't mind if we keep it, but maybe make it package private?